packages/cozy-harvest-lib/src/stories/AccountSelectorHeader.stories.ts

```diff
@@ -0,0 +1,24 @@
+import type { Meta, StoryObj } from '@storybook/react'
+
+import { AccountSelectorHeaderFixtures } from './AccountSelectorHeader.fixtures'
```

Comment:
I don't like these relative imports, can we get rid of them?

Reply:
> I don't like these relative imports

This is a personal opinion, and this is not very useful that way. If you have args for or against, it's way better.

Reply:
you're right it's not very important anyway. I'll use alias if they exist in the project otherwise it'll stay that way